src/setStateIndicator/randomNum.js

```diff
@@ -1,16 +1,8 @@
-export const getRandomNumber = (max) => {
-  return Math.floor(Math.random() * max);
-};
-
-export const setStateDependingRandomNum = (maxNum, setIndicator) => {
+export const setStateDependingRandomNum = (setIndicator) => {
   const time_between_run_function_getRandomNumber_ms = 5000;
   let timerId = setInterval(() => {
-    let randomNumber = getRandomNumber(maxNum);
-    if (randomNumber === 1) {
-      setIndicator("sick");
-      clearInterval(timerId);
-      return;
-    }
+    let randomNumber = Math.random() > 0.2 ? null : setIndicator("sick");
```

Comment:
instead of null, you could provide a noOp function, meaning that this function is called but does not do anything. It's ok to call a function that does not do anything, because it's cheap to do this in a JS engine.
A noOp function is something like `const noOp = function() {}` or your could google how people suggest writing it.

Reply:
done